docs/cmd/tkn_pipelinerun_describe.md

```diff
@@ -34,6 +34,7 @@ or
   -L, --last                          show description for last PipelineRun
       --limit int                     lists number of PipelineRuns when selecting a PipelineRun to describe (default 5)
   -o, --output string                 Output format. One of: json|yaml|name|go-template|go-template-file|template|templatefile|jsonpath|jsonpath-file.
+  -S, --status                        use status to get the status of a PipelineRun
```

Comment:
should we add for tasks also just like steps

Reply:
Sure, with the `--tasks` flag?